webpack/assets/javascripts/foreman_toast_notifications.js

```diff
@@ -0,0 +1,40 @@
+import * as ToastActions from './react_app/redux/actions/toasts';
+import store from '../javascripts/react_app/redux';
+import $ from 'jquery';
+/* eslint-disable max-params  */
+
+export function notify(message, type, link, sticky) {
```

Comment:
they will be undefined if not sent, which will be evaluated the same as false or null, i cant see any reason to use default values here.